WooCommerce/Classes/ViewRelated/Hub Menu/HubMenuViewModel.swift

```diff
@@ -325,12 +329,23 @@ extension HubMenuViewModel {
     }
 
     struct Subscriptions: HubMenuItem {
-        static var id = "upgrades"
+        static var id = "subscriptions"
 
         let title: String = Localization.subscriptions
         let description: String = Localization.subscriptionsDescription
         let icon: UIImage = .shoppingCartPurpleIcon
         let iconColor: UIColor = .primary
+        let accessibilityIdentifier: String = "menu-subscriptions"
+        let trackingOption: String = "upgrades"
```

Comment:
I adjusted the `Subscriptions` menu item to contain the same tracking option as the `Upgrades` one since they're essentially the same action but with different presentations. WDYT?